python/test/unit/language/test_core.py

```diff
@@ -5965,6 +5965,28 @@ def kernel(Out):
     assert h.asm["ptx"].count("%smid") == 1
 
 
+def test_memrealtime(device):
```

Comment:
can we unify it with the existing test rather than duplicating it. Looks almost exactly the same

Reply:
Sure, I'll see what I can do.